### PR TITLE
Update RELAP5-3D instructions

### LINKED ISSUE #24905: Update Relap5-3D instructions
## Reason
Update the Relap5-3D instructions

## Design
Create a markdown page to produce desired effects. Move away from templates for this specific application that is not exactly a MOOSE-based application.

## Impact
Instructions will be more specific to Relap5-3D

### PR DESCRIPTION
**Pull request by milljm**
Initial changes requested to RELAP5-3D instructions.

Closes #24905

### CODE REVIEW

File: modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md [hunk 1]
```diff
@@ -1,20 +1,238 @@
...
+!alert! tip title=Copying files from remote HPC machine to your machine
+Copying files from an HPC cluster to your machine first requires that you follow the instructions:
+[inl/hpc_remote.md]. This workflow is for advanced users, comfortable with modifying their SSH
+settings, and familiar with their terminal in general.
+
+Example commands you would use with a terminal opened on your machine to copy files from HPC:
 
 ```bash
-relap5 --help
+scp <your hpc user id>@hpclogin:~/testing/input_out.e /path/to/the/local/destination
```
Comment (milljm) [inline modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md:236]: Is the name of this file accurate?

File: modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md [hunk 2]
```diff
@@ -1,20 +1,240 @@
...
+
+1. Under the Project label, enter the project from the projects page for which you will be
+  performing your work. Contact the HPC staff for help determining which project your work pertains
+  to.
+
+1. Under the Cluster label, select your choice of machine. (Sawtooth or Lemhi)
+
+1. Under the Job Type label, select Login.
+
+1. Click Launch
```
Comment (lindsayad) [inline modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md:26]: This number is weird but I guess it numbers correctly on the web-page?!

File: modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md [hunk 3]
```diff
@@ -1,20 +1,240 @@
...
+0031201 0,7.0+6,9.78293+5,2.58184+6,0,0,2
+0031301 0,0,0,1
+* reactivity table
+20201100 reac-t
+20201101  0.0,0.0, 1.1,1.1  2.0,-2.0
+* reactor kinetics input
+30000000 point
+30000001 no-gamma  1.0+6  0.0  200.0
+30000011 11
+. end of case
```
Comment (lindsayad) [inline modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md:98]: Are we allowed to put this stuff up?

File: modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md [hunk 4]
```diff
@@ -1,20 +1,240 @@
...
+mkdir ~/testing
+```
+
+You will need to use a terminal editor to create the file, such as emacs, vi, or vim. Please search
+the internet for how to use these tools, if unfamiliar.
+
+Alternatively, you can create the file on your local machine using your favorite editor and copy it
+to the remote by running scp or rsync commands. However, doing so will cause your file to have
+carriage-return and line-feeds. These must be removed prior to running relap53D. To do so use the
+“dos2unix” linux command.
```
Comment (lindsayad) [inline modules/doc/content/ncrc/applications/ncrc_hpc_relap5.md:115]:
```suggestion
`dos2unix` linux command.
```

**Review by lindsayad — DISMISSED** [review]

**Review by lindsayad — DISMISSED** [review]

**Review by lindsayad — APPROVED** [review]

**Pull request merged**